### ISSUE TITLE
Resistance to TSH

### ISSUE THREAD

**Issue by euniceyi** [obophenotype/human-phenotype-ontology#4734]
This is a request to add an agnostic/NOS term **Resistance to thyroid-stimulating hormone** given that the currently most applicable term of **TSH receptor defect** implies a specific etiology that is not always known in observed phenotypes of resistance to TSH. Thus we recommend making **Thyroid-stimulating hormone receptor defect** a child of the new term **Resistance to thyroid-stimulating hormone** which in turn will be a child of **Abnormality of thyroid physiology** to accommodate phenotypes with serum levels implying TSH resistance but without known mechanism.

**Preferred term label**
Resistance to thyroid-stimulating hormone

**Synonym**
Resistance to thyrotropin

**Definition**
"Reduced sensitivity of thyroid follicle cells to stimulation by biologically active TSH."
PMID: 28648507

**Parent term**
Abnormality of thyroid physiology
https://hpo.jax.org/app/browse/term/HP:0002926

**Child term**
Thyroid-stimulating hormone receptor defect
https://hpo.jax.org/app/browse/term/HP:0011789

**Your nano-attribution (ORCID)**
0000-0001-6328-7035

**Comment by pnrobinson** [comment]
I see what you mean and this is a good suggestion. Are there other forms of resistance to TSH that we should list here while we are at it?

**Comment by euniceyi** [comment]
Hi Dr. Robinson,

The main known forms of of Resistance to TSH (RTSH) are:
1) TSH receptor gene defect
2) defect in PAX8, transcription factor required for both normal differentiated function and growth of thyroid cells

I'm unsure if the _PAX8_ mutation qualifies as a phenotype. Please advise on whether it would be an appropriate new term in HPO, and I will submit a full ticket.

Worth noting is that according to the cited 2017 review (PMID: 28648507), only a small proportion of RTSH patients have been found to have mutations in either _TSHR_ or _PAX8_, and most etiologies are presumed to be unknown. So the most essential change for this case would seem to be adding the general category of Resistance to TSH and subsuming the receptor defect within that. 

Thank you for your feedback!

**Comment by nicolevasilevsky** [comment]
@pnrobinson I can add this term, and the TSH receptor gene defect term, as @euniceyi suggested

**Comment by pnrobinson** [comment]
I think this is actually pathophysiology and not the actual phenotype?
"Reduced sensitivity of thyroid follicle cells to stimulation by biologically active TSH."
What is actually being measured? The production of T3 or T4?

**Comment by euniceyi** [comment]
I agree that the definition I proposed does sound like pathophysiology. My understanding is that Resistance to TSH is considered a syndrome of reduced sensitivity to biologically active TSH, and the  phenotypic finding is elevated TSH but no goiter (so perhaps the definition could be that instead of my original suggestion: "Elevated TSH in the absence of goiter"). 

Patients can have normal or decreased T4 depending on whether mutations result in uncompensated TSH resistance, compensated TSH resistance, or nonclassic TSH resistance (with paradoxically increased thyroidal iodine uptake).

Again, my main concern is that most patients with elevated TSH are not found to have the known TSH receptor defect, so adding more of an umbrella term that would parent the specific mechanistic term that is currently in place (TSH receptor defect) would allow a more accurate term for the majority phenotype of TSH resistance.

**Comment by pnrobinson** [comment]
I think that we can address this in the same was as  #4734, I will wait for your response there. Thanks for the input!

**Comment by pnrobinson** [comment]
Implemented the changes above. Thanks! This will be visible online with the next HPO release in about one month.
